ISSUE_TEMPLATE.md

```diff
@@ -2,16 +2,17 @@
 *Please read the contribution guidelines first!*
 
 *Feel free to use the following template. Be as detailed as possible.*
+*Please see the text preview to avoid unnecessary formatting errors.*
 *Don't forget to remove this text before submitting.*
 
 ----
 
 ### Checklist
 
 - [x] This is a bug report.
-- [ ] This is a plugin request.
 - [ ] This is a feature request.
-- [ ] I used the search function to find already opened/closed issues or pull requests.
+- [ ] This is a plugin (improvement) request.
+- [ ] I have read the contribution guidelines.
```

Comment:
Let's link to the contribution guidelines here so people don't have to search for them.